src/unstract/sdk/index.py

```diff
@@ -115,7 +117,13 @@ def index_file(
         x2text_adapter_inst: X2TextAdapter = x2text.get_x2text(
             adapter_instance_id=x2text_adapter
         )
-        extracted_text = x2text_adapter_inst.process(input_file_path=file_path)
+        extract_file_path = None
+        if not is_summary:
+            directory, filename = os.path.split(file_path)
+            extract_file_path: str = os.path.join(
+                directory, "extract", os.path.splitext(filename)[0] + ".txt"
```

Comment:
@chandrasekharan-zipstack `extract` is a separate directory which not part of file name.

Reply:
@chandrasekharan-zipstack I also have a cleaner approach to get the extracted_file_path with `pathlib.Path`. I will update it when we make changes to handle Path with `pathlib.Path` across the `unstract` repo.

Reply:
@Deepak-Kesavan if the cleaner approach can be integrated in the existing code base without a separate refactor - we can adopt that. Anyway since these changes are merged - we can look into it later